File: TypeScript/abc199_a/main.ts
async function getInputToNumberArray(): Promise<number[]> {
    const input = await new Response(Deno.stdin.readable).text();
    const lines = input.trim().split('\n');

    const numbers = lines[0].split(' ').map(Number);
    return numbers;
}

const [a, b, c] = await getInputToNumberArray();

function solve(a: number, b: number, c: number): string {
    return a * a + b * b < c * c ? 'Yes' : 'No';
}

console.log(solve(a, b, c))

export{};

File: Python/abc199_a/main.py
def solve(a, b, c):
    if a * a + b * b < c * c:
        return "Yes"
    else:
        return "No"

def main():
    a, b, c = map(int, input().split())
    print(solve(a, b, c))

if __name__ == "__main__":
    main()
